core/src/main/java/org/opentosca/toscana/core/transformation/execution/ExecutionTask.java

```diff
@@ -72,7 +73,10 @@ private void serveArtifact() {
 
     private void transform() {
         try {
-            plugin.transform(new TransformationContext(transformation, csarContentDir, transformationRootDir));
+            TServiceTemplate template = transformation.getCsar().getTemplate().orElseThrow(() ->
+                new IllegalStateException("TServiceTemplate is null, will not start transformation."));
```

Comment:
Maybe reword this to "TServiceTemplate of csar {} is null, could not start transformation"

Reply:
fixed